spec.html

```diff
@@ -1024,6 +1024,7 @@ <h1>Mathematical Operations</h1>
       <emu-note>
         <p><emu-eqn>floor(_x_) = _x_ - (_x_ modulo 1)</emu-eqn>.</p>
       </emu-note>
+      <p>An <dfn>interval</dfn> from lower bound _a_ to upper bound _b_ is a possibly-infinite, possibly-empty set of numeric values of the same numeric type. A numeric value _x_ is included in the interval if it satisfies the criteria _a_ &lt; _x_ and _x_ &lt; _b_. If the lower bound is described as inclusive, _a_ &lte; _x_ is used instead of _a_ &lt; _x_. If the upper bound is described as inclusive, _x_ &lte; _b_ is used instead of _x_ &lt; _b_. An <dfn>inclusive interval</dfn> is an interval where both bounds are inclusive. *NaN* is never included in an interval. An interval with an exclusive bound of *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub> includes neither *+0*<sub>𝔽</sub> nor *-0*<sub>𝔽</sub>. An interval with an inclusive bound of *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub> includes both *+0*<sub>𝔽</sub> and *-0*<sub>𝔽</sub>.</p>
```

Comment:
We never actually use "interval" without specifying inclusive/exclusive for both ends (usually by the now-defined term "inclusive interval"); as such, I'd prefer we avoid defining what "interval" means when used without specifying the ends.

For example:

```diff
- A numeric value _x_ is included in the interval if it satisfies the criteria _a_ &lt; _x_ and _x_ &lt; _b_. If the lower bound is described as inclusive, _a_ &lte; _x_ is used instead of _a_ &lt; _x_. If the upper bound is described as inclusive, _x_ &lte; _b_ is used instead of _x_ &lt; _b_.
+ A numeric value _x_ is included in the interval if it satisfies both the lower bound &le; _x_ / a &lt; _x_ (for inclusive/exclusive lower bounds, respectively) and the upper bound _x_ &le; _b_ / x &lt; _b_ (for inclusive/exclusive upper bounds, respectively).
```

Reply:
Eh, I still like my formulation better. "interval" is still useful in practice to link to all half-open intervals, even if there are no fully open intervals. Also, apparently unqualified "interval" implies "open interval" to mathematicians and I'd prefer to respect that intuition if there's relatively little downside.

Reply:
> Also, apparently unqualified "interval" implies "open interval" to mathematicians

No it doesn't. Different mathematicians use different customs (in analysis, for example, if you encounter an unqualified "interval" it is almost certainly meant to be closed), and most people will specify open or closed when using the term. We should always specify and should foreclose the possibility of using the term without specifying.

Reply:
Alright, reworded. Let me know what you think.